### PR TITLE
Update cumulative list of changes from second to third edition

### PR DESCRIPTION
**Pull request by svgeesus**
This needs to be up to date, so the implementation report can demonstrate that each substantive change is tested.

### CODE REVIEW

**Comment by digitaltvguy** [comment]
> This needs to be up to date, so the implementation report can demonstrate that each substantive change is tested.

Who has this list?

**Comment by chrisn** [comment]
The list is [in the spec](https://w3c.github.io/PNG-spec/#F-ChangeList). We can update it based on the [git commit history](https://github.com/w3c/PNG-spec/commits/main/).

File: index.html [hunk 1]
```diff
@@ -7712,36 +7712,67 @@ <h3 id="changes-20031110">Changes since the <a href="https://www.w3.org/TR/2003/
...
+        <p>Added the <a class="chunk" href="#cICP-chunk">cICP</a> chunk, Coding-independent code points for video signal type
+          identification, to contain image format metadata defined in [[ITU-T-H.273]] which enables PNG to contain [[ITU-R-BT.2100]] High Dynamic Range
+          (<a>HDR</a>) and Wide Color Gamut (WCG) images.</p>
+      </li>
 
-      <li>The three previously defined, but unofficial, chunks for APNG have been added. This brings the PNG specification
-      into alignment with widely deployed industry practice.
+      <li>
+        <p>The previously defined <a class="chunk" href="#eXIf">eXIf</a> chunk has been moved from the PNG-Extensions document
+        [[PNG-EXTENSIONS]] into the main body of this specification, to reflect it's increasing use.</p>
```
Comment (chrisn) [inline index.html:7737]:
```suggestion
        [[PNG-EXTENSIONS]] into the main body of this specification, to reflect its increasing use.</p>
```

**Comment by chrisn** [comment]
Should we include the addition of the color chunk precedence table?

**Comment by svgeesus** [comment]
> Should we include the addition of the color chunk precedence table?

Yes, we should. And [there is a test for it](https://wpt.fyi/results/png/cICP-wins.html?label=master&label=experimental&aligned) to check `cICP` wins over `iCCP`.

**Review by chrisn — APPROVED** [review]

**Pull request merged**